### PR TITLE
update caption

### PR DESCRIPTION
**Pull request by vvaths**
## this is a PR
- bullet 1
- bullet 2

### CODE REVIEW

File: _slides/01-intro.md [hunk 1]
```diff
@@ -3,4 +3,4 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vvaths) [inline _slides/01-intro.md:5]: inline comment!

File: _slides/01-intro.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ layout: slide
...
 
-Use the right arrow to begin!
+Welcome to the amazing slide!
```
Comment (vvaths) [inline _slides/01-intro.md]:
```suggestion
Welcome to the amazing slide! Hello!
```

**Review by vvaths — COMMENTED** [review]
self review!!

File: _slides/01-intro.md [hunk 3]
```diff
@@ -3,4 +3,4 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Use the right arrow to begin!
+Welcome to the amazing slide! Hello!
```
Comment (vyas000) [inline _slides/01-intro.md]:
```suggestion
Welcome to the amazing slide! Hello World!
```

Comment (vyas000) [reply _slides/01-intro.md]: good job!

**Review by vyas000 — CHANGES_REQUESTED** [review]
Please make the changes suggested!

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**